### DISCUSSION TITLE
Can we run inside MacOS?

### DISCUSSION THREAD

**Discussion by stivyw** [sickcodes/Docker-OSX#451] [General]
I'm trying run it inside MacOS Big Sur. It's not possible?

**Comment by sickcodes** [comment]
Not yet, pretty sure it's possible though. I don't have a Mac to test on 🌚

**Comment by wanyukang** [comment]
hola @stivyw, can you share your result to run in macOS? I try to do the same thing in my 2017-mbp and I got no luck, my cpu looks like supported kvm, and I installed qemu from brew successful, but I dont think my env init in the right way.
